Ensono/stacks-dotnet
# Feature: Implement swagger endpoints for menu operations

#### 📲 What

This PR includes the changes for:
- Swagger configuration for OpenAPi docs
- Endpoints specified in the OpenApi contract for Menu API
- Implement versioning in the swagger configuration

#### 🤔 Why
		
Implement the API operations specified in the API contract
		
#### 🛠 How
		
Added swagger dependencies
Created dummy endpoint that satisfies each operation specified in the API contracts

#### 👀 Evidence
		
Once deployed to an environment, the root of the api(i.e: http://dev.azure.amidostacks.com/api/menu) must redirect the user(browser) to a swagger page showing all operations available.

![image](https://user-images.githubusercontent.com/943026/60355674-e79e5b00-99c6-11e9-82bd-00dc72685d78.png)

		 
#### ✅ Acceptance criteria

- [ ] Meeting the following Coding Standard

#### 🕵️ How to test

- Deploy the PR to an environment:
- Navigate to the root endpooint of the menu api (i.e: http://dev.azure.amidostacks.com/api/menu)
- The browser should be redirect to the swagger page
- The operations specified in the OpenAPI spec should be visible in the swagger docs
